src/components/ArticleReaction/ArticleReaction.module.css

```diff
@@ -38,13 +38,14 @@
     padding-top: 5px;
   }
 
-  .popup button:last-child{
-    background-color: #fff;
-  }
-
   .transparentBtn {
     border: none;
     cursor: pointer;
     background: transparent;
     color: #FFF;
-  }
\ No newline at end of file
+  }
+
+
+.popup button:hover {
+  background-color: #eee; 
+}
```

Comment:
You may want to remove commented code on line 30, and add a new line at the end of this file